doc/tutorials/core/univ_intrin/univ_intrin.markdown

```diff
@@ -0,0 +1,337 @@
+
+    * 64-bit floats in 512 bit register:
+
+            v_float64x8 reg2                     // reg2.nlanes = 8
+
+
+### Load and Store operations
+
+Now that we know how registers work, let us look at the functions used for filling these registers with values.
+
+* **Load**: Load functions allow you to *load* values into a register.
+    * *Constructors* - When declaring a register structure, we can either provide a memory address from where the register will pick up contiguous values, or provide the values explicitly as multiple arguments (Explicit multiple arguments is available only for Constant Sized Registers):
+
+            float ptr[32] = {1, 2, 3 ..., 32};   // ptr is a pointer to a contiguous memory block of 32 floats
+
+            // Variable Sized Registers //
+            int x = v_float32().nlanes;          // set x as the number of values the register can hold
+
+            v_float32 reg1(ptr);                 // reg1 stores first x values according to the maximum register size available.
+            v_float32 reg2(ptr + x);             // reg stores the next x values
+
+            // Constant Sized Registers //
+            v_float32x4 reg1(ptr);               // reg1 stores the first 4 floats (1, 2, 3, 4)
+            v_float32x4 reg2(ptr + 4);           // reg2 stores the next 4 floats (5, 6, 7, 8)
+
+            // Or we can explicitly write down the values.
+            v_float32x4(1, 2, 3, 4);
+
+    <br>
+    * *Load Function* - We can use the load method and provide the memory address of the data:
+
+            float ptr[32] = {1, 2, 3, ..., 32};
+            v_float32 reg_var;
+            reg_var = vx_load(ptr);              // loads values from ptr[0] upto ptr[reg_var.nlanes - 1]
+
+            v_float32x4 reg_128;
+            reg_128 = v_load(ptr);               // loads values from ptr[0] upto ptr[3]
+
+            v_float32x8 reg_256;
+            reg_256 = v256_load(ptr);            // loads values from ptr[0] upto ptr[7]
+
+            v_float32x16 reg_512;
+            reg_512 = v512_load(ptr);            // loads values from ptr[0] upto ptr[15]
+
+        @note The load function assumes data is unaligned. If your data is aligned, you may use the `vx_load_aligned()` function.
+    <br>
+
+* **Store**: Store functions allow you to *store* the values from a register into a particular memory location.
+    * To store values from a register into a memory location, you may use the *v_store()* function:
+
+            float ptr[4];
+            v_store(ptr, reg); // store the first 128 bits(interpreted as 4x32-bit floats) of reg into ptr.
+<br>
+@note Registers containing values of a type may lead to wrong interpretation during storage. For example: if **ptr** is a pointer to a 32-bit float and reg is a register containing 32-bit integers,  the values in ptr may not be as intended. You may cast the registers into the proper type before carrying out operations.
```

Comment:
`v_store` defined so the type compatibility check will be performed. Actually it's impossible to store integer values to float pointer without additional treatment. One would have to either v_reinterpret register values or type cast the pointer

Reply:
@note If the pointer and the register have different datatype, you can typecast the pointer before calling `v_store`, or store the values in the in a different data-type and then use the `v_reinterpret_` functions to convert the registers into proper type.

Is this fine?

Reply:
Reinterpret and pointer casting could lead to wrong interpretation(if this reinterpretation isn't intended). I mean type correctness is checked here. So one have to either use proper types or do special treatment to mark casting intended.
 Maybe reduce notice to `Ensure **ptr** has the same type as register or cast  the register into the proper type before carrying out operations.`?